### ISSUE TITLE
"You broke the flow!" Appears after every action.

### ISSUE THREAD

**Issue by Capiron** [HabitRPG/habitrpg-chrome#170]
I broke the flow, apparently. Every time the timer changes from Pomodoro to break and/or back again, I lose health. I'm also having the "constantly losing health" bug. My character is really taking an unfair beating; I don't know if I can still use this extension if this continues.

**Comment by Joseph67843** [comment]
Yes, this is the same as #156 and #165. Unfortunately, the devs are currently occupied with the main site, so this is probably on hold unless a volunteer steps up. The tomatoes integration works a little better - it only increments the combo pomodoro task, but it doesn't take away points.
